spec/presenters/family_presenter_spec.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe FamilyPresenter do
+  let(:user) { create(:user) }
```

Comment:
Why do you create a `user` in family presenter? Why can't you create `family`?

Reply:
refactored